docs/alert/contact_points.md

```diff
@@ -43,7 +43,7 @@ To use SMTP with a PMM Docker installation:
     *NB: If you are using your Gmail’s SMTP credentials as shown above, you will have to generate an app password and fill it in as the value of your $GF_SMTP_PASSWORD variable.*
 2. Pass in the `.env` file to Docker run using the `--env-file` flag:
     ```
-    docker run --env-file=.env -p 443:443 -p 80:80 percona/pmm-server:2
+    docker run --env-file=.env -p 443:443 -p 80:80 pmm-server:3.0.0-beta
```

Comment:
```suggestion
    docker run --env-file=.env -p 443:443 -p 80:80 perconalab/pmm-server:3.0.0-beta
```

Reply:
Cat, we need put the namespace (aka org) everywhere, otherwise it's not gonna work.